studygroups/templatetags/extras.py

```diff
@@ -30,3 +31,7 @@ def is_organizer(user):
     """
     return user.is_staff or TeamMembership.objects.filter(user=user, role=TeamMembership.ORGANIZER)
 
+@register.filter
+def category_title(id, categories):
```

Comment:
I'm not a big fan of template filters, but I guess this is okay. When template filters get used a lot, it easily hides a lot of logic and dependencies. In this case it seems like you're just looking up data in a dictionary that came from the Discourse API, so that's fine.